sdk/monitor/monitor-opentelemetry/test/internal/unit/shared/config.test.ts

```diff
@@ -453,14 +453,20 @@ describe("Library/Config", () => {
       const config = new InternalConfig();
       config.azureMonitorExporterOptions.connectionString =
         "InstrumentationKey=1aa11111bbbb1ccc8dddeeeeffff3333";
-      assert.ok(warnStub.calledOn, "warning was raised");
+      // passes
```

Comment:
Shows what I mean in https://github.com/Azure/azure-sdk-for-js/pull/32819/files#diff-716c78a6b3caa3ff2df2fdc016071e643c06bb9ee849bcf25a21ae8fca958215R286-R306

Note: GH can be slow to load and jump to the correct file, you could either wait a bit for it to scroll or navigate to the file directly 👍